test/jdk/javax/swing/JScrollBar/4865918/bug4865918.java

```diff
@@ -34,16 +34,20 @@
 import javax.swing.JFrame;
 import javax.swing.JScrollBar;
 import javax.swing.SwingUtilities;
+import java.awt.event.MouseAdapter;
 import java.awt.event.MouseEvent;
+import java.util.concurrent.CountDownLatch;
 
 import java.util.Date;
 
 public class bug4865918 {
 
     private static TestScrollBar sbar;
     private static JFrame frame;
+    private static volatile CountDownLatch latch;
 
     public static void main(String[] argv) throws Exception {
+        latch = new CountDownLatch(1);
```

Comment:
Initialise the latch at its declaration.
```suggestion
    private static final CountDownLatch latch = new CountDownLatch(1);

    public static void main(String[] argv) throws Exception {
```

`mousePressLatch` would be a more descriptive name.